.eleventy.js

```diff
@@ -186,6 +190,18 @@ async function transformMarkup(rawContent, outputPath) {
       extraOutputSubdirectory: this.extraOutputSubdirectory,
     };
 
+    if (addNoScript) {
+      Array.from(dom.window.document.getElementsByTagName('img')).forEach(
```

Comment:
Have you considered using the existing `images` variable here instead of `dom.window.document.getElementsByTagName('img')`?

Users might not expect this plugin to wrap all `img` elements with a `noscript` if they have customised the `imgSelector` config.

Reply:
i updated the pr with the variable